### PR TITLE
Merge master to Release 1.11 branch

### PR DESCRIPTION
**Pull request by jayanthvn**
**What type of PR is this?**
Merge master to release-1.11 branch

**Which issue does this PR fix**:
N/A

**What does this PR do / Why do we need it**:
Merges latest master PRs to release 1.11 branch

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

Yes testing will be done before release 1.11.3
**Automation added to e2e**:

Yes

**Will this PR introduce any new dependencies?**:

No

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
No

**Does this change require updates to the CNI daemonset config files to work?**:

No

**Does this PR introduce any user-facing change?**:

No
```release-note
Release 1.11.3
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jayanthvn** [comment]
Release branch 1.11 is "This branch is [9 commits ahead](https://github.com/aws/amazon-vpc-cni-k8s/compare/master...release-1.11), [37 commits behind](https://github.com/aws/amazon-vpc-cni-k8s/compare/release-1.11...master) master." 

1 extra commit is the final merge.

**Comment by orsenthil** [comment]
Please trigger the integration test change this PR before merge. (Make sure the changes are picked up by having the correct base branch).

**Comment by jayanthvn** [comment]
Yes the integration test is already running.

**Comment by orsenthil** [comment]
Thank you, Jayanth. :)

File: .github/workflows/stale_issue_pr.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: 'Stale issue & PR handler'
```
Comment (vikasmb) [inline .github/workflows/stale_issue_pr.yaml:1]: Do we need stale issue & PR handler activated in release branch as well ? Is there a way to restrict this workflow to master branch only ?

Comment (vikasmb) [reply .github/workflows/stale_issue_pr.yaml:1]: Synced up offline, workflows are only driven from the files in master branch.

**Review by vikasmb — APPROVED** [review]
lgtm

**Comment by jayanthvn** [comment]
Integration test has passed.

**Pull request merged**